server.js

```diff
@@ -20,4 +24,81 @@ app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+
+//  Create a new message
+app.use(express.json());
+
+app.post("/messages", function (req, res) {
+  const newMsg = req.body;
+  console.log(req);
+  
+  // Level 2 - simple validation
+  const msg = {
+    id: idCount++,
+    from: req.body.from,
+    text: req.body.text,
+  };
+  if (!newMsg.from || !newMsg.text) {
+    res.status(400).send("Rejected: empty or missing text or from property.");
```

Comment:
this is good, your POST works well and good to see you thinking about what response to send if the request is invalid, 400 works well here